sentry-sample/build.gradle.kts

```diff
@@ -27,7 +27,7 @@ android {
             isMinifyEnabled = false
         }
         getByName("release") {
-            proguardFiles(getDefaultProguardFile("proguard-android-optimize.txt"), "proguard-rules.pro")
+            consumerProguardFiles("proguard-rules.pro")
```

Comment:
Do we need `consumerProguardFiles` on the application module? No one is going to consume its Proguard rules

Reply:
opss, it was supposed to be sentry-android, will fix it, good catch, thanks